tools/bumper/parser.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"reflect"
+
+	"github.com/pkg/errors"
+
+	"gopkg.in/yaml.v2"
+)
+
+type metadata struct {
+	Tag		string `yaml:"tag"`
+}
+
+type component struct {
+	Url         string `yaml:"url"`
+	Commit      string `yaml:"commit"`
+	Metadata 	metadata
+}
+
+type ComponentsList struct {
+	Multus component `yaml:"multus"`
+	LinuxBridge component `yaml:"linux-bridge"`
+	BridgeMarker component `yaml:"bridge-marker"`
+	Kubemacpool component `yaml:"kubemacpool"`
+	Nmstate component `yaml:"nmstate"`
+	OvsCni component `yaml:"ovs-cni"`
+	MacvtapCni component `yaml:"macvtap-cni"`
+}
+
+type ComponentsConfig struct {
+	Components ComponentsList
+}
```

Comment:
We don't need this to be public since now it's next to main at the same package
```golang
type componentsConfig struct {
	components Components
}
```

Reply:
Done